File: POOJAVA/src/clasesPOO/Usuarios.java
package clasesPOO;

public class Usuarios {
    String idUser;
    String nUser;
    String email;
    String clave;
    String estado;
    public Usuarios(String user,String emai)
    {
        establecerDatos(user,emai);
    }
    public String ObtenerUser()
    {
       return nUser;
    }
    public String ObtenerEmail()
    {
       return email;
    }
    void  establecerDatos(String usr,String ema)
    {
        nUser=usr;
        email=ema;
    }
}

File: POOPHP/app/Model/Usuarios.php
<?php
namespace App\Model;
class Usuarios
{
   public $idUser="";
   public $nUser="";
   public $email="";
   public $clave="";
   public $estado="";
   public function __construct(String $user,String $emai)
    {
        $this->establecerDatos($user,$emai);
    }
    public function ObtenerUser()
    {
        return $this->nUser;
    }
    public function ObtenerEmail()
    {
        return $this->email;
    }
    public function establecerDatos(String $usr,String $ema)
    {
        $this->nUser=$usr;
        $this->email=$ema;
    }
}
?>